README.md

```diff
@@ -42,6 +42,32 @@ php artisan translatable:export es,bg,de
 
 The command with the "es,bg,de" parameter passed will create es.json, bg.json, de.json files with translatable strings or update the existing files in the `resources/lang` folder of your project.
 
+### Persistent strings
+
+Some strings are not included in the export, because they are being dynamically generated. For example:
+
+```{{ __(sprintf('Dear :name, your order has been %s', $orderStatus)) }}```
```

Comment:
@lasmanis Can we do the same in the usual way?
```
{{ __('Dear :name, your order has been :status', ['name' => '....', 'status' => $orderStatus]) }}
```

Reply:
Not if you want the 'status' value to be translated as well.

Sometimes you could get away with doing something like:

```
@switch($orderStatus)

    {{-- Option A: One string per status value --}}

    @case('approved')
        {{ __('Dear :name, your order has been approved', ['name' => $name]) }}
        @break

    {{-- OR Option B: One main translatable string PLUS separate status translations --}}

    @case('cancelled')
        {{ __('Dear :name, your order has been :status', ['name' => $name, 'status' => __('cancelled')] }} 
        @break
    ...
@endswitch
```
The above code can be handled by the default functionality, without needing the persistent-strings implementation.

But, the resulting code is less clean and also, does not help if you want to change the syntax (eg. the word order) depending on the value of 'status'.

Reply:
Thanks @lasmanis. I see now. Could you please then improve the example by passing 'name' argument to the `__()`? Something like I did in my example or just get rid of `:name` completely to not confuse the reader.

Reply:
@lasmanis I mean, the `:name` token doesn't have any value int the example, so I think the string can be just `"Dear Bob,  ..."`.

Reply:
Yes, I understand what you mean. I added `:name` in the example because I wanted to point out the difference between a dynamic value that does not need translation (name) and one that does (status).

But, I agree, there's no point at providing an example that makes things more confusing, instead of less 😃 I'll remove it.

Thank you @kkomelin for your efforts and for your time! Much appreciated!!!

Reply:
> I'll remove it.

Perfect @lasmanis. Thanks. 

> Thank you @kkomelin for your efforts and for your time! Much appreciated!!!

You're welcome!